.sass-lint.yml

```diff
@@ -0,0 +1,18 @@
+rules:
```

Comment:
Hum linting for me is something really important (and should be mandatory into all serious companies...), the .sass-lint.yml provide a decent example and config for scss.